kategory/src/main/kotlin/kategory/data/SetKW.kt

```diff
@@ -0,0 +1,77 @@
+package kategory
+
+@higherkind
+@deriving(Monad::class, Traverse::class, MonoidK::class)
```

Comment:
`Set` can't have neither `Monad` nor `Traverse` instances. Just `Foldable` and `MonoidK` because sets don't guarantee order of iteration and that is why `flatMap` gives you back an Iterable. We should only have those two mentioned instances